backend/src/main/java/com/example/backend/OrderingSysService.java

```diff
@@ -102,17 +103,29 @@ public List<DishInCart> getAllDishesInCart() {
        return dishInCartRepo.findAll();
     }
 
-    public Order creatOrder() {
+    public void deleteDishInCart(String id) {
+       dishInCartRepo.deleteById(id);
+    }
+
+    public DishInCart changeQuantity(DishInCartDTO dishInCartDTO){
+        int dishId = dishInCartDTO.getDishId();
+        List<DishInCart> allByDishIdIs = dishInCartRepo.findAllByDishIdIs(dishId);
+        DishInCart selectd = allByDishIdIs.get(0);
```

Comment:
spelling thing? is it selected or selectedDish? For your Code-Companions it's nice to know exactly what this is about :)